packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -23,32 +23,25 @@ export async function getZrxTradeQuote(
   if (buyAsset.chain !== ChainTypes.Ethereum || sellAsset.chain !== ChainTypes.Ethereum) {
     throw new ZrxError('getQuote - Both assets need to be on the Ethereum chain to use Zrx')
   }
-  if (!sellAmount && !buyAmount) {
-    throw new ZrxError('getQuote - sellAmount or buyAmount amount is required')
+  if (!sellAmount) {
+    throw new ZrxError('getQuote - sellAmount is required')
   }
 
   const useSellAmount = !!sellAmount
   const buyToken = buyAsset.tokenId || buyAsset.symbol
   const sellToken = sellAsset.tokenId || sellAsset.symbol
 
-  const { minimum: minQuoteSellAmount, maximum: maxSellAmount } = await getZrxMinMax(input)
+  const { minimum, maximum } = await getZrxMinMax(buyAsset, sellAsset)
 
-  const minQuoteSellAmountWei = bnOrZero(minQuoteSellAmount).times(
+  if (!minimum) throw new ZrxError('getQuote - sellAmount is required')
```

Comment:
Same here, I can't see how `minimum` could be falsy - it comes from `bnOrZero(1).dividedBy(bnOrZero(usdRate)).toString()`.

Whilst we are here, `bnOrZero(1)` 😕

Reply:
agree